### DISCUSSION TITLE
Non-blurry fonts when rendering at a low resolution

### DISCUSSION THREAD

**Discussion by jhgarner** [bevyengine/bevy#11443] [Q&A]
I've got a font that should match a pixel art aesthetic (https://kottke.org/plus/type/silkscreen/index.html), but when I try to draw a Text2DBundle with it, the text comes out blurry. For sprites I've seen that you can use `ImagePlugin::default_nearest()` to disable anti-aliasing, but I couldn't find an equivalent setting for Text. Is there a way to render text such that it snaps to the pixel grid without anti-aliasing in Bevy? Alternatively, are there ways to work with positions and sizes so the font won't anti-alias?

![Screenshot from 2024-01-20 15-51-05](https://github.com/bevyengine/bevy/assets/7400280/51cbdae0-4a64-4e48-bfb5-ced66f087e7c)

**Comment by jakub-pietras** [comment]
@jhgarner I was struggling with the same issue, and I didn't find any way to turn off the antialiasing for font. But I managed to find a roundabout way to get good results. The blurred antialiasing comes mostly from zoom at a low pixel count (your font is zoomed in like your game), so you can multiply your font size few times, and then use `Transform` `scale` property to take it down in size. I also found out I had to keep the multiplication and division value even, so working with 2, 4... to avoid subpixel sizes. That means you are effectively rendering it at a bigger size and then downscaling to greatly reduce the effect of antialiasing. Wish we could just disable it though :(

Before (font size 20, scale 1):
![image](https://github.com/bevyengine/bevy/assets/43368580/89f904f1-e12a-4485-8681-0022cd5fe021)

After (font size 70, scale 0.25):
![image](https://github.com/bevyengine/bevy/assets/43368580/f069f067-14a0-4a46-97d7-15050dde88c2)
